### PR TITLE
Doc 72

### PR DESCRIPTION
**Pull request by StarfallProjects**
Resolves DOC-72

- Refactored file structure to sit better within Next
- First pass over filename & project structure guidelines

### CODE REVIEW

File: explorer/analyzer/file-formats.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Format: required. Stoplight supports `md`, `json`, `yaml` and `yml`.
+
+### Examples
+
+#### OpenAPI spec (v2 or v3):
+
+```
+# Without versioning
+my-api.oas.json
+my-api.oas.yaml
```
Comment (chohmann) [inline explorer/analyzer/file-formats.md:27]: I think we'll still allow them to not have version in their filenames. However, that means that spec and nodes within the spec will be un-versioned. Can't remember how un-versioned nodes show up in Explorer in dojo, but should probably mention this consequence.

File: explorer/analyzer/file-formats.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+# With versioning
+my-api.1-0.oas.json
+my-api.1-0.oas.yaml
+```
+
+#### JSON schema spec:
```
Comment (chohmann) [inline explorer/analyzer/file-formats.md]: maybe just call it JSON schema file? Not sure if spec is the right word for it.

File: explorer/analyzer/file-formats.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+#### OpenAPI spec (v2 or v3):
+
+```
+# Without versioning
+my-api.oas.json
+my-api.oas.yaml
+
+# With versioning
+my-api.1-0.oas.json
+my-api.1-0.oas.yaml
```
Comment (chohmann) [inline explorer/analyzer/file-formats.md]: forgot the `v`. So `my-api.1-0.oas.json` --> `my-api.v1-0.oas.json`

File: explorer/analyzer/file-formats.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+#### JSON schema spec:
+
+```
+# Without versioning
+my-api.schema.json
+
+# With versioning
+my-api.1-0.schema.json
```
Comment (chohmann) [inline explorer/analyzer/file-formats.md]: forgot the `v` here as well.

File: explorer/analyzer/file-formats.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+#### Markdown files
+
+Markdown files are source files for documentation. Do not use markdown to write your API specification.
+<!-- QUERY: is this ^ correct? -->
```
Comment (chohmann) [inline explorer/analyzer/file-formats.md]: this is correct.

**Review by chohmann — CHANGES_REQUESTED** [review]
This is awesome! Just a few suggestions.

File: explorer/analyzer/file-formats.md (hunk 1 above)
Comment (StarfallProjects) [reply explorer/analyzer/file-formats.md:27]: Added a warning up top

**Pull request merged**